packages/cloudflare/README.md

```diff
@@ -161,6 +161,15 @@ The following example automatically generates `_routes.json` while including and
   });
 ```
 
+### `imageService`
+
+`imageService: "passthrough" | "cloudflare"`
+
+Determines which image service is used by the adapter. The following options are available, if no option is provided the adapter will default to `passthrough ` mode when an incompatible image service is configured. Otherwise, it will use the globally configured image service:
```

Comment:
```suggestion
Determines which image service is used by the adapter. The adapter will default to `passthrough ` mode when an incompatible image service is configured. Otherwise, it will use the globally configured image service:
```